src/traces/pie/plot.js

```diff
@@ -37,28 +37,10 @@ module.exports = function plot(gd, cdpie) {
         var pieGroup = d3.select(this);
         var cd0 = cd[0];
         var trace = cd0.trace;
-        var tiltRads = 0; // trace.tilt * Math.PI / 180,
-        var depthLength = (trace.depth||0) * cd0.r * Math.sin(tiltRads) / 2;
-        var tiltAxis = trace.tiltaxis || 0;
-        var tiltAxisRads = tiltAxis * Math.PI / 180;
-        var depthVector = [
-            depthLength * Math.sin(tiltAxisRads),
-            depthLength * Math.cos(tiltAxisRads)
-        ];
-        var rSmall = cd0.r * Math.cos(tiltRads);
-
-        var pieParts = pieGroup.selectAll('g.part')
-            .data(trace.tilt ? ['top', 'sides'] : ['top']);
-
-        pieParts.enter().append('g').attr('class', function(d) {
-            return d + ' part';
-        });
-        pieParts.exit().remove();
-        pieParts.order();
```

Comment:
Man, I totally forgot that I had written all this crap for 3D pies, I thought #1152 was all of it - but now between this commit and https://github.com/plotly/plotly.js/pull/2117/commits/679ed9cf051eda44d53fa3aa3b6a031cd34ddab5 I think it's finally all gone.